std/internal/math/biguintcore.d

```diff
@@ -652,14 +652,14 @@ public:
     // If wantSub is false, return x + y, leaving sign unchanged
     // If wantSub is true, return abs(x - y), negating sign if x < y
     static BigUint addOrSubInt(Tulong)(const BigUint x, Tulong y,
-            bool wantSub, ref bool sign) pure nothrow if (is(Tulong == ulong))
+            bool wantSub, ref bool sign) pure nothrow @safe if (is(Tulong == ulong))
     {
         BigUint r;
         if (wantSub)
         {   // perform a subtraction
             if (x.data.length > 2)
             {
-                r.data = cast(immutable) subInt(x.data, y);
+                r.data = (() @trusted => cast(immutable) subInt(x.data, y))();
```

Comment:
since subInt returns an array allocated with `new`, this can be safely cast to immutable.

Reply:
Might want to put a note of that there.

Reply:
Ditto throughout